File: cpp/Leetcode/c++/problem448.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        map<int,int>m;
        set<int>s(nums.begin(),nums.end());
        vector<int>lst;
        for(int i=1;i<nums.size()+1;i++){
            m[i]=0;
        }
        for(auto i:s){
            m[i]+=1;
        }
        for(auto i:m){
            if(i.second==0){
                lst.push_back(i.first);
            }
        }
        return lst;

    }
};

File: cpp/Leetcode/Python/problem448.py
class Solution:
    def findDisappearedNumbers(self, nums):
        s=list(set(nums))
        s.sort()
        d={}
        lst=[]
        for i in range(1,len(nums)+1):
            d[i]=0
        for i in s:
            d[i]+=1
        for [k,v] in d.items():
            if(v==0):
                lst.append(k)
        return lst




        